pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -86,6 +86,30 @@ process prepareBusco {
     """
 }
 
+
+/**
```

Comment:
The workflow now includes a lot of comments describing what the processes do. Where those comments are a general description of the process, I'd suggest to move them to the Groovydoc of the process itself. Here is an illustration of what I have in mind:
```suggestion
/**
* Link annotations present in the cache.
*
```

Reply:
I would like to keep those comments close to the call of the process. I will consider adding a groovydoc for the workflow as the pipeline matures.